src/lib.rs

```diff
@@ -131,14 +131,14 @@ pub mod qdrant;
 mod auth;
 mod builder_ext;
 mod builder_types;
+pub mod builders;
```

Comment:
Does this have to be `pub`, it would change the public API interface too?

If without `pub` isn't good enough, maybe we can use `pub(crate)`.

Reply:
done https://github.com/qdrant/rust-client/pull/199/commits/779098315d2f8c5c8044f56f30954c569e0e969b